### PR TITLE
flake: minor changes

### PR DESCRIPTION
**Pull request by drupol**
This PR:

- [x] Update `flake.lock` to remove the warning
  image
- [x] Make a minor refactor of `flake.nix`

### CODE REVIEW

File: flake.nix [hunk 1]
```diff
@@ -2,32 +2,19 @@
   description = "Nix CycloneDX Software Bills of Materials (SBOMs)";
 
   inputs = {
-
```
Comment (nikstur) [inline flake.nix:5]: I'd like to keep the whitespace.

File: flake.nix [hunk 2]
```diff
@@ -2,32 +2,19 @@
...
-      url = "github:edolstra/flake-compat";
-      flake = false;
-    };
-
-    flake-parts = {
-      url = "github:hercules-ci/flake-parts";
-      inputs.nixpkgs-lib.follows = "nixpkgs";
-    };
-
+    flake-compat.url = "github:nix-community/flake-compat";
```
Comment (nikstur) [inline flake.nix:7]: The nix-community one seems to be less maintained than eelcos right now.

**Review by nikstur — COMMENTED** [review]
A few small thing but looks good otherwise. Thank you!

File: flake.nix (hunk 1 above)
Comment (drupol) [reply flake.nix]: Fixed.

File: flake.nix (hunk 2 above)
Comment (drupol) [reply flake.nix]: Fixed

**Review by nikstur — APPROVED** [review]
Edited my few nits myself. Thanks!

**Comment by drupol** [comment]
Why is flake-utils needed?

**Comment by nikstur** [comment]
> Why is flake-utils needed?

So that we only have one version both in rust-overlay and in pre-commit-hooks-nix

**Pull request merged**